### PR TITLE
Feat/edit profile modal

### PR DESCRIPTION
**Pull request by nikaakin**

### CODE REVIEW

File: tailwind.config.js
Comment (MilosIl) [inline tailwind.config.js:1]: our design team doesn't know what values, so we can work around it with tailwind measurement. use something that is close by.

I found some margins that are 39 and other is 41 on same axis..

**Review by MilosIl — COMMENTED** [review]
Good implementation on mui + tailwind

**Review by MilosIl — APPROVED** [review]

File: src/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -1,11 +1,213 @@
...
+                    </RadioGroup>
+                  </FormControl>
+                )}
+              />
+              <Button
+                type='submit'
+                className='font-jomhuria text-center w-full gap-10 font-light cursor-pointer pt-1 px-4 outline-none rounded-2xl bg-[#F52B38] text-4xl text-white bg-opacity-90 hover:bg-opacity-100 transition-opacity'
+              >
+                Save
+              </Button>
```
Comment (Ben2oid) [inline src/pages/Profile/Profile.tsx:205]: Only thing I'm wondering if you don't fill out the form, should you be able to submit the data? Nonetheless good job. Works really nice and looks good too.

File: src/pages/Profile/Profile.tsx [hunk 2]
```diff
@@ -1,11 +1,213 @@
...
+                  register={register('phone_number')}
+                  name='phone_number'
+                  placeholder='Phone number'
+                />
+              </div>
+              <Controller
+                name='birth_date'
+                control={control}
+                render={({ field: { onChange, value, ref } }) => (
+                  <LocalizationProvider dateAdapter={AdapterDayjs}>
```
Comment (Ben2oid) [inline src/pages/Profile/Profile.tsx:127]: I'm wondering if there is a calendar type that let's you choose the month the same way you can set the years. Some calendars let you choose the year and the month then. It's just more user friendly.

**Review by Ben2oid — APPROVED** [review]

**Comment by Ben2oid** [comment]
I'm approving anyways, just added comments and thoughts to some of the features.

**Pull request merged**